python/cudf/cudf/tests/test_series.py

```diff
@@ -1346,3 +1346,17 @@ def test_nullable_bool_dtype_series(data, bool_dtype):
     gsr = cudf.Series(data, dtype=bool_dtype)
 
     assert_eq(psr, gsr.to_pandas(nullable=True))
+
+
+@pytest.mark.parametrize("data", [([0, 1, 2, 3])])
+def test_series_transpose(data):
+    psr = pd.Series(data=data)
+    csr = cudf.Series(data=data)
+
+    cudf_transposed = csr.transpose()
+    pd_transposed = psr.transpose()
+    cudf_property = csr.T
+
+    assert_eq(pd_transposed, cudf_transposed.to_pandas())
+    assert_eq(pd_transposed, cudf_property.to_pandas())
```

Comment:
```suggestion
    assert_eq(pd_transposed, cudf_transposed)
    assert_eq(pd_transposed, cudf_property)
```

You usually shouldn't need to explicitly convert to pandas before comparing because `assert_eq` does that internally.